controllers/withwatcher/kyma_controller_test.go

```diff
@@ -119,8 +121,8 @@ func registerDefaultLifecycleForKymaWithWatcher(kyma *v1beta1.Kyma, watcher *v1b
 		By("Deleting watcher CR")
 		Expect(controlPlaneClient.Delete(suiteCtx, watcher)).To(Succeed())
 		By("Ensuring watcher CR is properly deleted")
-		Eventually(isWatcherCrDeletionFinished(client.ObjectKeyFromObject(watcher)), Timeout, Interval).
-			Should(BeTrue())
+		Eventually(isWatcherCrDeletionFinished, Timeout, Interval).WithArguments(watcher).
+			WithOffset(4).Should(Succeed())
```

Comment:
what's the purpose of this magic `WithOffset(4)` here? is it just skip 4 times, then start to verify the value? If this test eventually should success, please remove this offset.